fighthealthinsurance/sentry_filters.py

```
@@ -1,32 +1,176 @@
+    """
+    if isinstance(value, str):
+        return value
+    try:
+        # Both of these run user-supplied code on a non-str: `not value` calls
+        # __bool__/__len__ and str() calls __str__. Either can raise, so both
+        # live inside the try -- outside it, the raise would propagate and
+        # discard the event.
+        if not value:
+            return ""
+        return str(value)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)